server/main.tf

```diff
@@ -47,7 +47,7 @@ provider "aws" {
 
 # data "archive_file" "lambda_greetings_server" {
 #   type        = "zip"
-#   source_dir  = "${path.module}/lambda"
+#   source_dir  = "${epath.modul}/lambda"
```

Comment:
This doesn't look right.

Reply:
@basilv  The I am not using the main.tf for any modules configurations for now.

Reply:
@explorertolu That's not the point. It's a very clear typo.

Reply:
@DerekRoberts I see the clear typo now, it wasn't obvious to me.

Thanks.